src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const CART_DB = './cartdb.json';
```

Comment:
I see all these constans are not used anywhere, do you need them or they can be removed?